lexicon/tests/providers/integration_tests.py

```diff
@@ -10,17 +9,20 @@
 from lexicon.providers.base import Provider as BaseProvider
 
 
-# Configure VCR. Parameter record_mode depends on the LEXICON_LIVE_TESTS environment variable value.
-record_mode = 'new_episodes' if os.environ.get(
-    'LEXICON_LIVE_TESTS', 'false') == 'true' else 'none'
+# Configure VCR. Parameter record_mode depends
+# on the LEXICON_LIVE_TESTS environment variable value.
+record_mode = 'none'
+if os.environ.get('LEXICON_LIVE_TESTS', 'false') == 'true':
+    record_mode = 'new_episodes'
```

Comment:
I wonder if its possible to do a check of the existing Cassettes and warn the user if the files already exist. In general we want clean cassettes with only one set of recordings.

Reply:
Do you mean with the `new_episodes` mode ? Because during the `none` mode, no cassette will be written, and if no cassette correspond to the test, or a cassette gives invalid request replay, an error will be thrown.
For `new_episodes`, if I recall correctly, the relevant cassette will be reused if it exists, and exceptions will be thrown also if the request replay does not correspond. If no cassette exists, then it is written. So with that approach, cassettes would not be rewritten without explicit action (deleting manually the cassette before).

Reply:
In fact, the behavior I describe is the `once` mode. The mode `new_episodes` will always rewrite the cassette. 

`once` became the default mode in recent versions of vcrpy. I think it would be relevant to do it also in lexicon.

Reply:
yeah, thats what I meant, `once` is probably the correct solution here. During development a developer may be creating changes that modify the HTTP requests, and should be deleting the cassettes for their provider sporadically. Only the actually used HTTP requests should be included in the final PR.

Reply:
All right, I pushed a commit to use once mode for live tests in this PR.